src/pages/Cart/components/HorizontalCard/HorizontalCard.jsx

```diff
@@ -0,0 +1,30 @@
+import { AiOutlineMinus, AiOutlinePlus} from "../../../../assets/icons"
+import { Button } from "../../../../Components/index"
```

Comment:
`import { Button } from "../../../../Components"
`

This will also work

Reply:
It's auto imported